pages/order.vue

```diff
@@ -166,4 +259,4 @@
     cursor: none;
     pointer-events: none;
   }
-</style>
```

Comment:
The code is good here (I'll nitpick some things, but... overall, quite good.)  

The file is getting a bit long, however.

I have a couple abstractions in mind.
* Install the Composition API library and put the price calculation into a reactive object in a composition function (my specific technique here might be a bit unusual (Gregg didn't recognize it), and I haven't screencasted about it, so I may end up doing this one myself).
* Putting the pricing table into a single-use component
* Making a reusable radio button component that abstracts the Vuetify component further

Do those component abstractions sound like a good idea to you?  And are there any others you would suggest?

Reply:
Update: Could you make the radio buttons into a button group instead?  https://vuetifyjs.com/en/components/button-groups/